### ISSUE TITLE
Word of the Day App

### ISSUE THREAD

**Issue by ManjotS** [KhalisFoundation/sttm-web#1460]
<div> Freshdesk Ticket ID: <a href="https://khalis.freshdesk.com/a/tickets/5009">5009</a></div>
                       <div> Freshdesk Ticket Agent: Aman Singh</div>
                       <div> Freshdesk Ticket Agent Email: saintsoldierx@gmail.com</div>
                       <div> Ticket Priority: Low</div>
                       <div> Freshdesk Ticket Description: </div><div dir="ltr">
<div>WaheguruJiKaKhalsaWaheguruJiKiFathe,</div>
<div><br></div>
<div>Firstly, thank you for all the great you guys do and have done. I have used this probably since late 2000s. I am amazed at how far it has come, and grateful to be able to connect to the Guru anywhere, anytime.<br>
</div>
<div>Anyway, I was wondering if you guys could have a 'Word of the day' option from the dictionary on the site and or in the app.<br>
</div>
<div>And if it is not too much, have an RSS option for it too.</div>
<div>Thank you.</div>
</div>

**Comment by ManjotS** [comment]
<div>Agent Aman Singh linked Freshdesk ticket <a href="https://khalis.freshdesk.com/a/tickets/5009">5009</a> for this issue.</div>

**Comment by gurjit03** [comment]
@ManjotS @saintsoldierx How about this feature? Do we wants to continue this one ?

**Comment by saintsoldierx** [comment]
Yes @gurjit03 this is a good suggestion, I think this would be a good widget for the front page
